CONTRIBUTING.md

```diff
@@ -1,34 +1,36 @@
-# Filing an issue
+# How to contribute
+
+## Filing an issue
 
 If you want your issue to be resolved quickly, please include in your
 issue:
 
 * Gatsby version, node.js version, OS version
 * The contents of your `gatsby-node.js`, `gatsby-browser.js`, and `package.json`.
 
-# Contributing
+## Contributing
 You can install the latest `master` version of Gatsby by following these
 simple steps:
 
 * Clone the repo, navigate to its directory.
-* Execute `npm install` to install packages.
-* Execute `npm uninstall -g gatsby && npm link`
+* Install Lerna if you don't have it already: `yarn global add lerna@^2.0.0-beta.0`
+* Execute `yarn && lerna bootstrap` to install dependencies and cross link all the packages.
 * Use `git pull` to update to latest Gatsby.
 
-Test suite can be run via `npm test`.
+Test suite can be run via `yarn test`.
 
 This project uses [FlowType](https://flowtype.org/) for static type checking.
 
 The usual contributing steps are:
 
 * Fork the [official repository](https://github.com/gatsbyjs/gatsby).
 * Clone your fork: git clone `git@github.com:<your-username>/gatsby.git`
-* Make sure tests are passing for you: npm install && npm test
-* Create a topic branch: git checkout -b topics/new-feature-name
-* Run `npm run watch` to watch code and compile your changes.
-* Clone one of the official repositories and "link" your fork of Gatsby
-  to it (run `npm link gatsby`).
+* Install dependencies: `yarn && lerna bootstrap`
+* Make sure tests are passing for you: `lerna run test`
+* Create a topic branch: `git checkout -b topics/new-feature-name`
+* Run `yarn build && yarn watch` to watch code and compile your changes.
```

Comment:
What I was seeing was if there were build errors, watch was completely silent, while build would actually build and show the errors. So it looked on my machine that watch didn't actually build if there were errors, and you couldn't see these errors, so I more wanted those to be front and centre for people if they're just getting started.

Also, `yarn watch` is actually equivalent (today at least) to `yarn run watch`, but I'm happy to change that over. It's true that if they do ever create a true `yarn watch` proper then this would suddenly stop working, while `yarn run watch` would continue to work.